### PR TITLE
Corrected run script, readme and added husky tasks

### PR DESCRIPTION
**Pull request by FrankIglesias**
## Summary 
- Corrected readme previsualization
- Added `_package.json` linter tastks
- Corrected `run.sh`, added yarn bin verification, node version verification and git init to create de hooks

### CODE REVIEW

**Review by mvbattan — APPROVED** [review]
👏

File: run.sh [hunk 1]
```diff
@@ -36,6 +44,7 @@ yarn global add package-json-merge > /dev/null 2>&1
 package-json-merge package.json packageB.json > packageA.json
 mv packageA.json package.json
 rm packageB.json
+git init
```
Comment (mvbattan) [inline run.sh:47]: I'd add a TODO comment for repository integration

Comment (FrankIglesias) [reply run.sh:47]: There will be no need :wink:

**Review by mvbattan — APPROVED** [review]
😭👏

**Pull request merged**